src/Umbraco.Infrastructure/Packaging/PackageDataInstallation.cs

```diff
@@ -1010,12 +1017,13 @@ private T FindContentTypeByAlias<T>(string contentTypeAlias, IContentTypeBaseSer
         /// </summary>
         /// <param name="dataTypeElements">Xml to import</param>
         /// <param name="userId">Optional id of the user</param>
+        /// <param name="entityContainersInstalled">List of entity containers installed by the package to be populated with those created in installing data types.</param>
         /// <returns>An enumerable list of generated DataTypeDefinitions</returns>
-        public IReadOnlyList<IDataType> ImportDataTypes(IReadOnlyCollection<XElement> dataTypeElements, int userId)
+        public IReadOnlyList<IDataType> ImportDataTypes(IReadOnlyCollection<XElement> dataTypeElements, int userId, List<EntityContainer> entityContainersInstalled)
```

Comment:
This is a breaking change, we need to old signature reintroduced and obsoleted

Reply:
Have restored the old signature.